_posts/2024-11-20-cockpit-329.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Cockpit 329
+author: mvo
+date: '2024-11-20'
+tags: cockpit, files
+slug: cockpit-329
+category: release
+summary: Multi host warnings and File creation'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from cockpit 329 and cockpit-files 12:
+
+
+## Shell: Extra warnings when connecting to remote hosts
+
+Connecting to multiple hosts in a single Cockpit session allows all
+these hosts to access each other freely. [Cockpit 322](https://cockpit-project.org/blog/cockpit-322.html) already
+deprecated this functionality on newer operating system releases.
+
+![screenshot of extra warnings when connecting to remote hosts]({{ site.baseurl }}/images/329-shell-extra-warnings-when-connecting-to-remote-hosts.png)
+
+
+This warning can be disabled by including the following in `/etc/cockpit/cockpit.conf`:
+
+```
+[Session]
+WarnBeforeConnecting=false
+```
+
+## Files: File creation support
+
+Cockpit now allows the creation of new files directly in the File
+browser.
+
+![screenshot of file creation support]({{ site.baseurl }}/images/329-cockpit-files-file-creation-support.png)
```

Comment:
It's super silly that someone would use this as a TODO note, but the example works. :+1: :rofl: